plugins/woocommerce-admin/client/marketplace/components/content/content.tsx

```diff
@@ -1,34 +1,120 @@
 /**
  * External dependencies
  */
-import { useContext } from '@wordpress/element';
+import { useContext, useEffect, useState } from '@wordpress/element';
+import { useQuery } from '@woocommerce/navigation';
 
 /**
  * Internal dependencies
  */
 import './content.scss';
+
+import { Product, SearchAPIProductType } from '../product-list/types';
+import { MARKETPLACE_SEARCH_API_PATH, MARKETPLACE_HOST } from '../constants';
+import { getAdminSetting } from '../../../utils/admin-settings';
 import Discover from '../discover/discover';
 import Extensions from '../extensions/extensions';
+import SearchResults from '../search-results/search-results';
 import Themes from '../themes/themes';
 import { MarketplaceContext } from '../../contexts/marketplace-context';
 
-const renderContent = ( selectedTab?: string ): JSX.Element => {
-	switch ( selectedTab ) {
-		case 'extensions':
-			return <Extensions />;
-		case 'themes':
-			return <Themes />;
-		default:
-			return <Discover />;
-	}
-};
-
 export default function Content(): JSX.Element {
 	const marketplaceContextValue = useContext( MarketplaceContext );
+
+	const [ productList, setProductList ] = useState< Product[] >( [] );
```

Comment:
I like to keep var names consistent with the parameter names in function signatures. The linter doesn't complain about var name duplication when I do this. I would also change the name of the `tab` param in `renderContent` to `selectedTab`. Please ignore if you disagree.

```suggestion
	const [ products, setProducts ] = useState< Product[] >( [] );
```